packages/x-bus/tsconfig.esm.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "esnext",
+    "outDir": "./dist/esm",
+    "declaration": false,
+    "declarationDir": null,
+    "noEmit": false
```

Comment:
I think this should inherit the value of the extended parent `tsconfig.json`, as we don't want to emit declaration files twice (here and in the `cjs` build).

```suggestion
```

Reply:
The `noEmit` is not only related to types but to [all compiler output files](https://www.typescriptlang.org/tsconfig#noEmit) so if we enable the `noEmit` flag we will not get the esm build 😅

Reply:
you're right, I always mix it with `types` emission 🤦🏿